extensions/lifecycle/conductor/LifecycleConductor.js

```diff
@@ -95,9 +96,34 @@ class LifecycleConductor {
         // - some entries are expired for each listing
         this._accountIdCache = new AccountIdCache(this._concurrency);
 
+        const blacklist = (this.lcConfig.conductor.filter && this.lcConfig.conductor.filter.deny) || {};
```

Comment:
There was a push a few years ago around switching from the terms whitelist and blacklist to alternatives ( see https://9to5google.com/2020/06/12/google-android-chrome-blacklist-blocklist-more-inclusive/ ). Just something to consider. Also you could do: `this.lcConfig.conductor.filter?.deny || {}`

Reply:
Unfortunately, optional chaining is not supported yet on this version.